### PR TITLE
Enable testing on Drupal 8.6.x

### PR DESCRIPTION
**Pull request by pfrenssen**
Relates to #54 and #53

### CODE REVIEW

**Comment by pfrenssen** [comment]
Added testing on 8.6.x, they are currently failing because of #53. Once we get these green under 8.6.x we should remove the `allow_failures` section.

File: .travis.yml [hunk 1]
```diff
@@ -19,13 +19,16 @@ env:
...
-    - TEST=PHPUnit
+    - TEST=8.5.x
+    - TEST=8.6.x
 
 matrix:
   fast_finish: true
   exclude:
     - php: 7.2
       env: TEST=PHP_CodeSniffer
+  allow_failures:
```
Comment (claudiu-cristea) [inline .travis.yml]: Shouldn't we add a TODO referring the issue where we remove this?

Comment (pfrenssen) [reply .travis.yml]: Well we are not sure yet, there might be other dragons lurking below the surface. But I will create a dedicated issue to track this.

Comment (claudiu-cristea) [reply .travis.yml]: We have the 8.6.x issue. That would remove this

**Comment by claudiu-cristea** [comment]
Looks good, merging.

**Pull request merged**